.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Active File",
+      "runtimeExecutable": "pnpm",
+      "args": ["tsx", "${file}"],
+      "cwd": "${workspaceFolder}",
+      "console": "integratedTerminal",
+      "sourceMaps": true,
+      "nodeVersionHint": 20
+    }
+  ]
+}
```

Comment:
This looks useful - how does it work?

Reply:
I use this to run the example TS files more easily.  Just open any example file, then in VS Code, go to the debugger (Cmd+Shift+D), then select "Launch Active File".  It'll run the program, and you can add breakpoints in the file:

<img width="1103" alt="image" src="https://github.com/lgrammel/modelfusion/assets/8092406/66756411-e834-4cc5-8737-a2e3ad9c4f1f">